src/backend/base/langflow/api/v1/schemas.py

```diff
@@ -281,6 +283,11 @@ class VertexBuildResponse(BaseModel):
     timestamp: datetime | None = Field(default_factory=lambda: datetime.now(timezone.utc))
     """Timestamp of the build."""
 
+    @field_serializer("data")
+    def serialize_data(self, data: ResultDataResponse) -> dict:
+        truncated_data = truncate_long_strings(data)
```

Comment:
If data is a `ResultDataResponse` it won't be truncated because we don't have for `BaseModel` in the `truncate_long_strings` function

Reply:
yess you are right.
i'm now checking the dump model to ensure data will be truncate if fits the requirements..